x-pack/plugins/lens/public/trigger_actions/open_lens_config/action.test.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import type { IEmbeddable } from '@kbn/embeddable-plugin/public';
+import { ActionExecutionContext } from '@kbn/ui-actions-plugin/public';
+import { overlayServiceMock } from '@kbn/core-overlays-browser-mocks';
+import { themeServiceMock } from '@kbn/core-theme-browser-mocks';
+import type { LensPluginStartDependencies } from '../../plugin';
+import { createMockStartDependencies } from '../../editor_frame_service/mocks';
+import { DOC_TYPE } from '../../../common/constants';
+import { ConfigureInLensPanelAction } from './action';
+
+describe('open config panel action', () => {
```

Comment:
"Edit filters" button still opens the vis in Lens

<img width="300" alt="Screenshot 2023-07-06 at 10 16 57" src="https://github.com/elastic/kibana/assets/1415710/32b7e8dc-7256-4da5-9c58-a3a51e3d8c8b">

Reply:
Nice catch, I will remove the edit button for the text based languages.